configure.ac

```diff
@@ -184,10 +185,10 @@ fi
 ##############################################################################
 # GNU Scientific Library
 
-AC_CHECK_FUNC(cblas_cgemm, [], [AC_CHECK_LIB(gslcblas, cblas_cgemm)])
-
-AC_CHECK_LIB(gsl, gsl_sf_bessel_Jn, [],
-	[AC_MSG_WARN([Missing GNU GSL library...Bessel-function field initialization will not be supported.])])
+AC_CHECK_LIB([m],[cos])
+AC_CHECK_LIB([gslcblas],[cblas_dgemm])
```

Comment:
Thanks for looking into this. The difficulty seems to be related to harminv. Your test program above works for me when built via the command line you suggested. But if I attempt to link the harminv library it fails:

```
% gcc -o foo foo.c -lgsl -lharminv -lopenblas
...
/home/homer/codes/lib/libgsl.so: undefined reference to `cblas_dgemm'
...
```

If I position `-lharminv` after `-lgsl -lopenblas` on the command line then it works:

```
% gcc -o foo foo.c -lgsl -lopenblas -lharminv
% foo 1 3.4
0.179226
```

I'm looking in to why this is happening, but thought I would post just this much in case somebody already knows what's going on.